test_conformance/api/test_queries.cpp

```diff
@@ -380,7 +381,8 @@ int check_get_command_queue_info_params(cl_device_id deviceID,
         clGetDeviceInfo(deviceID, host_queue_query, sizeof(host_queue_props),
                         &host_queue_props, NULL);
     test_error(error, "clGetDeviceInfo failed");
-    log_info("CL_DEVICE_QUEUE_ON_HOST_PROPERTIES is %d\n", host_queue_props);
+    log_info("CL_DEVICE_QUEUE_ON_HOST_PROPERTIES is %" PRId64 "\n",
```

Comment:
` cl_queue_properties` is a typedef of `cl_properties`, which is a typdef of `cl_ulong`, so `PRIu64` might be better.

Reply:
Thanks for spotting that, fixed.